crates/api/src/arc.rs

```diff
@@ -0,0 +1,83 @@
+//! Kitsune2 arcs represent a range of hash locations on the DHT.
+//!
+//! An arc can exist in its own right and refer to a range of locations on the DHT.
+//! It can also be used in context, such as an agent. Where it represents the range of locations
+//! that agent is responsible for.
+
+/// The definition of an arc for Kitsune.
+///
+/// The lower bound is inclusive, the upper bound is exclusive.
+pub type ArcLiteral = (u32, u32);
+
+/// A full arc is represented by `(0, u32::MAX)`.
+pub const ARC_LITERAL_FULL: ArcLiteral = (0, 0);
```

Comment:
Doc is now wrong. Also, I did this with the upper bound being inclusive for the logic in the peer store because that was more intuitive to me. We need to either change that logic or change this logic.

Reply:
also see https://github.com/holochain/kitsune2/pull/47#issuecomment-2524252635

Reply:
Superseded by arc PR